README.md

```diff
@@ -384,6 +384,11 @@ $ kubectl get pod etcd-cluster-0000 -o yaml | grep "image:" | uniq
 
 Check the other two pods and you should see the same result.
 
+## Namespace
```

Comment:
can we move this to the top of the readme file? Namespace is related to the deployment of operator, which is the first thing users should understand.